handlers/handlers.go

```diff
@@ -110,3 +115,72 @@ func GeographyRender(rend RenderClient) http.HandlerFunc {
 		return
 	}
 }
+
+//GeographyListpageRender ...
+func GeographyListpageRender(rend RenderClient) http.HandlerFunc {
+
+	return func(w http.ResponseWriter, req *http.Request) {
+		var page geographyListPage.Page
+
+		vars := mux.Vars(req)
+		areaTypeID := vars["areaTypeID"]
+
+		resp, err := http.Get(`https://api.dev.cmd.onsdigital.co.uk/v1/code-lists/` + areaTypeID + `/editions/2016/codes`)
+		fmt.Println("error err", err)
+		fmt.Println("error resp.StatusCode", resp.StatusCode)
+		if err != nil {
+			setStatusCode(req, w, err)
+			return
+		}
+		if resp.StatusCode == 404 {
+			fmt.Println("ERROR 404 not 500")
```

Comment:
not clear what this block is doing - are we turning 404s into 500s? also presumably `err` is `nil` at this point since we've just got past the `if err != nil {` check